source/drivers/null/ur_null.h

```diff
@@ -0,0 +1,36 @@
+/*
+ *
+ * Copyright (C) 2019-2021 Intel Corporation
+ *
+ * SPDX-License-Identifier: MIT
+ *
+ * @file ur_null.h
+ *
+ */
+#pragma once
```

Comment:
I think we should use the `#ifndef` pattern here.

Reply:
Done